stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java

```diff
@@ -187,8 +210,25 @@ public static void dispatchCloseMessage(WebSocketOpenConnectionInfo connectionIn
         bValues[1] = new BInteger(closeMessage.getCloseCode());
```

Comment:
Shall we handle the case where close status code is -1? We cannot pass it to the resource. It should be 1006 if it is an abnormal closure.

Reply:
done https://github.com/wso2/transport-http/pull/148